Blocks/URL/url.h

```diff
@@ -515,6 +515,23 @@ struct URLPathArgs {
 
   void add(const std::string& arg) { args_.push_back(arg); }
 
+  std::string ComposeURLPathFromArgs() const {
+    std::string result;
+    for (auto cit = args_.crbegin(); cit != args_.crend(); ++cit) {
+      result += "/" + *cit;
```

Comment:
`EncodeURLComponent`?

Reply:
But `URLWithoutParametersParser::ComposeURL()` doesn't do it. Consistency FTW?

Reply:
One could argue `EncodeURLComponent` must be there for both. What if there's a slash in one of the URL path args?